php/php-xdebug/Portfile

```diff
@@ -15,12 +15,18 @@ master_sites            ${homepage}/files/
 php.branches            5.3 5.4 5.5 5.6 7.0 7.1 7.2 7.3
 php.extensions.zend     xdebug
 
-if {[vercmp ${php.branch} 7.0] >= 0} {
+if {[vercmp ${php.branch} 7.1] >= 0} {
     version             2.8.0
     revision            0
     checksums           rmd160  0fc3a106fa03cbc3b6b2384cf7f1ed89f975b31b \
                         sha256  cb1d117ecbec7409a408cf24e59d92f7e3816fe8e2331dd0ffc3dce60228c264 \
                         size    238122
+} elseif {[vercmp ${php.branch} 7.0] >= 0} {
+    version             2.7.2
```

Comment:
the version is decreased here - if the port did not build before, this is fine. Otherwise it will need to increase the ```epoch``` value (i.e., add ```epoch 1```) to have it recognized as an update.

Reply:
php70-xdebug @2.8.0 did build. If it nevertheless does not work, then yes, in addition to the changes already made in this PR, the epoch will need to be increased, for this subport only.

Reply:
Thanks, that wasn't something I was aware of. I've now added it.